assignments/arrays.js

```diff
@@ -63,32 +63,57 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+for(let i = 0; i < inventory.length ; i++){
+    if (i === 33){
```

Comment:
```suggestion
    if (inventory[i].id === 33){
```
So here you are saying, if the index is equal to 33. We want the id that is 33. So look at my revision, it's saying, if the item in inventory at the index of i has the id of 33, then do stuff.